### PR TITLE
Jamie edit model style updates

### PR DESCRIPTION
**Pull request by jamiewaese-uncharted**
Lots of edits to the model edit operator drilldown....

**BEFORE**
Monosnap Terarium 2024-02-16 16-46-14

Monosnap Terarium 2024-02-16 16-47-35
AI assistant - before

------------------------------------------------
**AFTER**
image

image

AI assistant - after

### CODE REVIEW

File: packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-input.vue
Comment (YohannParis) [inline packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-input.vue:1]: This is a big nono, we cannot remove functionality like that, each notebook can handle multiple language and should be kept as such.
@mwdchang could you confirm or deny it?

File: packages/client/hmi-client/src/workflow/ops/model-edit/tera-model-edit.vue
Comment (YohannParis) [inline packages/client/hmi-client/src/workflow/ops/model-edit/tera-model-edit.vue:1]: Men... introducing a div soup just before the hackathon... this is pure evil!

**Review by YohannParis — CHANGES_REQUESTED** [review]

File: packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-input.vue
Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-input.vue:1]: I am confused as to the problem? 
We can send the `contextLanguage` and `contextLanguageOptions` as props to this component. If this is the case is there any issues?

This was intended but I have been looking at a few other things and so far everything is python so i shortcut it for now.

Comment (jamiewaese-uncharted) [reply packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-input.vue:1]: k, I restored the dropdown and moved the toggle switch into the component

File: packages/client/hmi-client/src/assets/svg/icons/message.svg
[the remark this answers is not in the document]
Comment (jamiewaese-uncharted) [reply packages/client/hmi-client/src/assets/svg/icons/message.svg:1]: good to know, but I'm trying to insert an icon into an input box that doesn't have a slot for it, and this was a quick and dirty work around

**Review by YohannParis — APPROVED** [review]

**Pull request merged**